elasticsearch-extensions/lib/elasticsearch/extensions/test/cluster.rb

```diff
@@ -299,23 +296,25 @@ def start
           def stop
             begin
               nodes = __get_nodes
-            rescue Exception => e
-              __log "[!] Exception raised when stopping the cluster: #{e.inspect}".ansi(:red)
+            rescue StandardError => e
```

Comment:
I agree its breaking but I think a good call as there are very few things that should ever rescue on an exception...much better to fail fast on things that are ruby exceptions rather than standard error.